rice-middleware/impl/src/main/java/org/kuali/rice/kew/actionlist/web/ActionListAction.java

```diff
@@ -650,7 +654,11 @@ protected PaginatedList buildCurrentPage(List<? extends ActionItemActionListExte
         form.setHasDisplayParameters(haveDisplayParameters);
 
         generateActionItemErrors(CUSTOMACTIONLIST_PROP, ACTIONLIST_BAD_CUSTOM_ACTION_LIST_ITEMS_ERRKEY, customActionListProblemIds);
-        return new PaginatedActionList(currentPage, actionList.size(), page, pageSize, "actionList", sortCriterion, sortOrder);
+        
+        if(export)
```

Comment:
This violates best practice, please ensure conditionals always have curly braces.